examples/shouldi/.gitignore

```diff
@@ -21,5 +21,7 @@ htmlcov/
 tests/golangci-lint-download/
 tests/javascript_algo-download/
 tests/npm-audit-download/
+tests/cargo-audit-download/
+tests/tarpaulin-download/
```

Comment:
If you could make another PR which puts all of these downloads under `tests/downloads` that would be good, so for example in that PR this line would be `tests/downloads/tarpaulin`

Reply:
This will allow us to put the downloads in the cache to speed up the CI tests. You will of course also need to change the paths in the calls to `cached_download_unpack_archive`

Reply:
sorry for being late, was stuck btw things. Do I need to add all the previous downloads in this file as under `tests/downloads` or only the new ones.